src/test/java/edu/kis/vh/nursery/RhymersJUnitTest.java

```diff
@@ -36,15 +36,12 @@ public void testIsFull() {
 			Assert.assertEquals(false, result);
 			ryhmer.countIn(888);
 		}
-
-		boolean result = ryhmer.isFull();
-		Assert.assertEquals(true, result);
 	}
 
 	@Test
 	public void testPeekaboo() {
 		DefaultCountingOutRhymer ryhmer = new DefaultCountingOutRhymer();
-		final int EMPTY_STACK_VALUE = -1;
+		final int EMPTY_STACK_VALUE = 0;
```

Comment:
Original constant should be used (it is better to use a constant from stacks interface).